app/js/controllers/reimburse-ctrl.js

```diff
@@ -0,0 +1,31 @@
+'use strict';
+
+angular
+    .module('app.controllers')
+    .controller('ReimburseCtrl', function($scope, $http) {
+      $scope.formFields = {};
+      $scope.submit = function() {
+
+        $http.post('https://api.tnyu.org/v3-test/expenses',{
```

Comment:
You can use the Restangular library here, it'll handle some of the logic for you (like the headers and the url). https://github.com/mgonto/restangular